cached_network_image/example/lib/main.dart

```diff
@@ -141,7 +141,7 @@ class BasicContent extends StatelessWidget {
           aspectRatio: 1.6,
           child: BlurHash(hash: 'LEHV6nWB2yk8pyo0adR*.7kCMdnj'),
         ),
-        imageUrl: 'https://blurha.sh/assets/images/img1.jpg',
+        imageUrl: 'https://blurha.sh/12c2aca29ea896a628be.jpg',
```

Comment:
I just updated this URL since the old `https://blurha.sh/assets/images/img1.jpg` didn't point to a resource anymore. You just got back a 404.